scaled_turbines_xy: 
- [69.70716510903318, 10.616353097966952]
- [52.18691588785004, -16.370554801133437]
- [43.88785046728864, 44.28633960415572]
- [26.552024922117, 20.609146824756657]
- [7.925233644859198, -5.613980662104623]
- [-8.304049844237777, -29.92765711836985]
- [-30.803738317758317, -53.159311323962896]
- [22.679127725855672, 80.37496408243376]
- [4.421183800622288, 54.08818822788528]
- [-13.283489096572353, 30.28369871311328]
- [-33.01682242990765, 4.0605712262515095]
- [-49.79937694704054, -19.489324817774616]
- [-69.16386292835001, -44.121243112471944]
- [-0.18940809968928818, 116.84547876683112]
- [-18.81619937694709, 91.19518658914832]
- [-38.7339563862939, 64.01733358698827]
- [-56.25420560747703, 40.21284407221627]
- [-74.51214953271042, 14.562551894533952]
